crates/hotshot/src/traits/election/static_committee.rs

```diff
@@ -16,7 +16,8 @@ use tracing::debug;
 
 /// Dummy implementation of [`Membership`]
 
-#[derive(Clone, Debug, Eq, PartialEq, Hash)]
+#[derive(Clone, Debug, Eq, PartialEq, Hash, Serialize, Deserialize)]
```

Comment:
I still don't understand why you're doing this, and I *really* hope you're not planning to send this over instead of the two parameters needed to construct it, as we previously discussed.

Reply:
No, I don't plan to send this over. Is it fine to keep the serde bound still?

Reply:
Removed serde bound. `pub trait Membership<TYPES: NodeType>` isn't serde